<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerDrrImageComputationRtkParametersWidget</class>
 <widget class="QWidget" name="qSlicerDrrImageComputationRtkParametersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RTK DRR Parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="Label_ForwardProjectionParameter">
       <property name="text">
        <string>Parameter:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ctkSliderWidget" name="SliderWidget_ForwardProjectionFilterParameter">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Fill pad pixels:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ctkSliderWidget" name="SliderWidget_FillPadPixels">
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QGroupBox" name="GroupBox_ForwardProjectionFilterType">
       <property name="title">
        <string>RTK forward projection filter</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QRadioButton" name="RadioButton_Joseph">
          <property name="text">
           <string>Joseph</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">ButtonGroup_ForwardProjectionFilter</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="RadioButton_JosephAttenuated">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>JosephAttenuated</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">ButtonGroup_ForwardProjectionFilter</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="RadioButton_Zeng">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Zeng</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">ButtonGroup_ForwardProjectionFilter</string>
          </attribute>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="RadioButton_CudaRayCast">
          <property name="text">
           <string>CudaRayCast</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">ButtonGroup_ForwardProjectionFilter</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="ButtonGroup_ForwardProjectionFilter"/>
 </buttongroups>
</ui>
